### PR TITLE
[risk=low][RW-9560] Replace the AAR completion banner when CT renewal is incomplete

### PR DESCRIPTION
**Pull request by jmthibault79**
Today when a user is directed to Controlled Tier renewal (e.g. they clicked the "time for renewal" banner) what they see is an Annual Renewal page which is focused on the Registered Tier, with a big banner at the top which uses the word "complete" twice.

Completion Banner

This is likely to be confusing!  The actual action to take here is to scroll down to the Controlled Tier section and renew the Controlled Tier training.

This PR changes the "complete" banner to one which directs the user to renew Controlled Tier modules, including a button which takes the user there directly.

Controlled_Tier_Renewal

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "react-onclickoutside": "^6.7.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
+    "react-router-hash-link": "^2.4.3",
```
Comment (jmthibault79) [inline ui/package.json:71]: React doesn't seem to be able to follow `#foo` links without this.  All my searching led me back to this... which seems to work fine.

File: ui/src/app/utils/access-utils.tsx [hunk 2]
```diff
@@ -495,19 +495,6 @@ export const isRenewalCompleteForModule = (status: AccessModuleStatus) => {
   );
 };
 
-export const isRtRenewalComplete = (profile: Profile): boolean => {
```
Comment (jmthibault79) [inline ui/src/app/utils/access-utils.tsx:508]: not used

File: ui/src/app/components/buttons.tsx [hunk 3]
```diff
@@ -325,6 +326,41 @@ export const ButtonWithLocationState = ({
   );
 };
 
+// uses HashLink to allow navigation to anchor IDs on the same page
+
+// example: if you have a component with id='user-info', then you can create a HashLinkButton which navigates to it:
+// <HashLinkButton path='#user-info'/>
+
+export const HashLinkButton = ({
```
Comment (jmthibault79) [inline ui/src/app/components/buttons.tsx:334]: a copy of our `Button` which uses `react-router-hash-link`.  Open to alternatives.

File: ui/src/app/pages/access/controlled-tier-card.tsx [hunk 4]
```diff
@@ -129,7 +129,11 @@ export const ControlledTierCard = (props: {
   const { enableComplianceTraining } = serverConfigStore.get().config;
 
   return (
-    <FlexRow data-test-id='controlled-card' style={styles.card}>
+    <FlexRow
+      id='controlled-card'
```
Comment (jmthibault79) [inline ui/src/app/pages/access/controlled-tier-card.tsx:133]: Adding this `id` lets us navigate to it via `#controlled-card`

File: ui/src/app/pages/access/data-access-requirements.tsx [hunk 5]
```diff
@@ -530,6 +546,28 @@ const Completed = () => (
...
+  <FlexRow
+    data-test-id='controlled-tier-renewal-banner'
+    style={styles.controlledRenewal}
+  >
+    <FlexColumn>
+      <div style={styles.controlledRenewalHeader}>
+        Controlled Tier Access Renewal
+      </div>
+      <div style={styles.controlledRenewalText}>
+        Please update your modules below.
```
Comment (jmthibault79) [inline ui/src/app/pages/access/data-access-requirements.tsx:545]: thoughts on this text?

File: ui/src/app/pages/access/data-access-requirements.spec.tsx [hunk 6]
```diff
@@ -140,27 +140,41 @@ describe('DataAccessRequirements', () => {
       ]
     );
 
-  const expireAllModules = () => {
```
Comment (jmthibault79) [inline ui/src/app/pages/access/data-access-requirements.spec.tsx:143]: renamed some things from "all" to "all RT" which is more accurate

File: ui/src/app/pages/access/data-access-requirements.spec.tsx [hunk 7]
```diff
@@ -1619,13 +1643,12 @@ describe('DataAccessRequirements', () => {
 
     await waitOneTickAndUpdate(wrapper);
 
-    expectComplete(wrapper);
-    expect(true).toEqual(true);
```
Comment (jmthibault79) [inline ui/src/app/pages/access/data-access-requirements.spec.tsx:1623]: removed a few of these... debugging leftovers?

**Review by evrii — APPROVED** [review]
Tested locally. Looks good to me.

**Pull request merged**